testapi/event/1/speakers

```diff
@@ -4,38 +4,41 @@
   "speakers": [
     {
       "biography": "Matthias Spielkamp is managing editor of iRights.info, covering legal, economic and policy aspects of digitisation (winner of the Grimme Online Award). As a founding partner at the iRights.Lab think tank, he advises public institutions, NGOs and private companies on online journalism, communication strategies and information security. Matthias held teaching assignments at various universities, is a frequent speaker at national and international conferences and testified several times before German parliamentary committees. He co-authored three books on journalism and copyright, serves on the board of the German chapter of Reporters without Borders (pro bono) and is a member of the American Council on Germany as a former McCloy Fellow. Recently, he curated the Groundbreaking Journalism event series.", 
+      "orglink": "en.rsf.org/",
       "country": "", 
-      "email": "", 
-      "facebook": null, 
+      "email": "ms@spielkamp.org", 
+      "facebook": "matthias.spielkamp", 
       "github": null, 
       "id": 1, 
-      "linkedin": null, 
+      "linkedin": "matthiasspielkamp", 
       "name": "Matthias Spielkamp", 
       "organisation": "Reporters without Borders", 
       "photo": "http://re-publica.de/sites/re-publica.de/files/styles/speaker_profile/public/media/pictures/120330-03538.jpg", 
       "position": null, 
       "sessions": [], 
-      "twitter": null, 
+      "twitter": "spielkamp", 
       "web": "http://re-publica.de/member/1004#profile-speaker"
     }, 
     {
       "biography": "Thom Nagy ist seit 2004 als Journalist t\u00e4tig. Ab 2006 schrieb er bei \u00ab20 Minuten\u00bb \u00fcber Popkultur bis er 2011 zur \u00abNeuen Z\u00fcrcher Zeitung\u00bb wechselte, wo er sich im NZZ Lab mit den Implikationen vernetzter Kommunikations-Strukturen auf das Mediensystem auseinander setzt. Neben seiner praktischen T\u00e4tigkeit doziert er am MAZ in Luzern zum Thema Web-Journalismus.", 
+      "orglink": "labs.nzz.ch/",
       "country": "", 
       "email": "", 
-      "facebook": null, 
+      "facebook": "ThomNagyMusic", 
       "github": null, 
       "id": 2, 
-      "linkedin": null, 
+      "linkedin": "thom-nagy", 
       "name": "Thom Nagy", 
       "organisation": "NZZ Lab", 
       "photo": "http://re-publica.de/sites/re-publica.de/files/styles/speaker_profile/public/media/pictures/thomnagy_pensive.jpg", 
       "position": null, 
       "sessions": [], 
-      "twitter": null, 
```

Comment:
This can be null. 
I appreciate you fixing this, but if your code assumes they are never null, it's not right. 
The server can give null values, and the frontend should be able to handle that.